gms/client/build.gradle

```diff
@@ -7,9 +7,9 @@ dependencies {
 
   dataModel project(':li-utils')
 
-  compile project(':metadata-dao')
+  compile externalDependency.gmaDaoApi
```

Comment:
Let's group `externalDependency` together?

Reply:
fyi this was done with `sed`, which is why things stayed on the same line.